### ISSUE TITLE
Can multifolderclone.py be used to from a TD to TD?

### ISSUE THREAD

**Issue by theghost970** [Spazzlo/folderclone#22]
Hi there!

Really awesome work, thank you for it.
Just a quick question though, the instructions are for copying from a shared folder to a Team Drive, but can I copy from a Team Drive to a Team Drive? If so, how?

Knowing that the SAa are added to both TDs.

**Comment by Mz49** [comment]
Thank you for dropping by! Very good question as well!

multifolderclone.py currently relies on the fact that all SAs can view the source folder, along with it's subfolders and files, and add files to the destination folder. So to answer your question, yes, you can copy from a shared drive to another shared drive. You'll just need to add all the SAs added to both, with the source folder with at least Viewer permissions, and in the destination folder with at least Contributor permissions. Same goes with any regular folder. As long as the SAs can view the source folder, they can clone from it as well! But that starts getting a little more complicated since you'll have to add all your SAs to that one folder.

tl;dr: Yes. Just make sure your SAs can view the source shared drive/folder.

**Comment by theghost970** [comment]
> Thank you for dropping by! Very good question as well!
> 
> multifolderclone.py currently relies on the fact that all SAs can view the source folder, along with it's subfolders and files, and add files to the destination folder. So to answer your question, yes, you can copy from a shared drive to another shared drive. You'll just need to add all the SAs added to both, with the source folder with at least Viewer permissions, and in the destination folder with at least Contributor permissions. Same goes with any regular folder. As long as the SAs can view the source folder, they can clone from it as well! But that starts getting a little more complicated since you'll have to add all your SAs to that one folder.
> 
> tl;dr: Yes. Just make sure your SAs can view the source shared drive/folder.

That's wonderful!
Thank a lot!
